apps/z2m_ikea_controller/z2m_ikea_controller.py

```diff
@@ -17,10 +17,10 @@
 
 
 def action(method):
-    def _action_impl(self, *args, **kwargs):
```

Comment:
We currently not use any keyword arguments, but is always a good practice to leave them as well as adding `*args`. So I would suggest to not delete it.

Reply:
Ok, I'm adding `**kwargs` back in the next commit